### PR TITLE
Pebble Update

### PR DESCRIPTION
**Pull request by therealDLondon**
## About the PR
- Added Prisoner, Reporter, and Librarian jobs
- Adjusted slots
- Changed some tiles and decals
- Removed powergamer loot
- Added some things to maints
- Removed virology
- Expanded medbay
- Expanded perma
- Added reporter shuttle
- Tweaked some lights
- Tweaked some wires
- Changed alot of windows
- Added a library
- Changed max population to 35 from 30
## Why / Balance
Pebble goes largely unmaintained, and has fallen behind in mapping standards

## Technical details
Changes map file and YAML

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: DLondon
- add: Added Reporter, Librarian, and Perma slots to Pebble!
- tweak: Adjusts various elements of Pebble Station

### CODE REVIEW

**Comment by deltanedas** [comment]
> reporter shuttle

mime shuttle when :trollface:

**Comment by deltanedas** [comment]
test fail is real, add a spawnpoint for reporter

**Comment by therealDLondon** [comment]
Trolled by spawns...

File: Resources/Prototypes/Maps/pebble.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@
           mapNameTemplate: '{0} Pebble Station {1}'
           nameGenerator:
             !type:NanotrasenNameGenerator
-            prefixCreator: 'NY'
+            prefixCreator: 'DV'
```
Comment (IamVelcroboy) [inline Resources/Prototypes/Maps/pebble.yml]: Don't change this. I want to keep the legacy of the server it was originally created on.

**Review by IamVelcroboy — APPROVED** [review]

**Pull request merged**